README.md

```diff
@@ -78,6 +78,33 @@ However, if you need to use the translate API, for example, you may need to chan
 ```bash
 lm_eval --model translator --tasks yanolja_translation --model_args api_key=[FRAGMA_API_KEY],target_lang=en,endpoint=https://fragma.prod.yanolja.in/pre/translate,model=deepl --task_config doc_to_text="{{source}}"
 ```
+
+### Puree Datasets
+Before using Puree dataset, you need to configure GCP credential for you machine. There are several options to do it:
+- Automatically through Google CLI ([Instruction](https://cloud.google.com/docs/authentication/provide-credentials-adc#google-idp))
+- Manually, by downloading service account credential:
+  - Download a service account credential from [here](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107823799478031745980/keys?organizationId=989139104574&project=special-tf-prod)
+  - Move the credential into `~/.config/gcloud/application_default_credentials.json`
+
+
+To use a Puree dataset on evaluation, you can pass Puree dataset URI on `dataset_path`. The URI has `puree://<dataset-id>` format, and you can find the `dataset-id` from Puree dataset detail page URL.
+
+```yaml
+# Original task with local dataset
+task: mdn_en-ko
+dataset_name: mdn_en-ko
+dataset_path: json
+dataset_kwargs:
+  data_files: ./data/MDN_translation_length_20_ko-en.jsonl
+include: translation_template_en-ko.yaml
+---
+# Updated task with puree dataset
+task: mdn_en-ko
+dataset_name: mdn_en-ko
+dataset_path: puree://ohgvC3elEgRFxvUXDkWe
```

Comment:
Can we use dataset file name for puree? it is hard to understand the source data.

Reply:
Unfortunately no, dataset name is not unique in Puree right now.
But you can always put a comment about the dataset name on the yaml.